### PR TITLE
refactor(kad): start initial bootstrap from behaviour.rs sooner + improve logs

### PR DESCRIPTION
**Pull request by ozwaldorf**
start the bootstrap sooner, currently getting an error bootstrap timed out with no more bootstraps remaining. This seems to fix that getting 8+ bootstraps remaining after the first one instead of none. 

Manually tested + unit tested, there are still some timeouts but the bootstrap keeps going. While the bootstrap are returning some odd peer ids (`1AdivKqEdvpxsBm1yQjTRYbqk7MdfVjRmQAzxm752XQVZ6` vs our peer ids which always look like: `12D3KooWN2GSWDQwtRWsgiYXDgCHX5scrdHQPman3tBH2WtyCyK3`), it seems to be working. during the time it took to get to the final bootstrap response, the node had peer connection events for 30+ new peers. I think some timeouts are from peers that had id's switch and are no longer available at the /p2p/<id> address that might still persist in some routing tables

tldr; improves boostrapping consistency by starting it sooner, and new peers are flooding in.

### CODE REVIEW

**Comment by b0xtch** [comment]
Does this improve consistency or solve the issue

**Review by b0xtch — APPROVED** [review]

**Comment by kckeiks** [comment]
This is interesting considering that the code that started the initial bootstrap is now only a few lines above. Does this change allow more time to avoid timeouts?

LGTM btw just curious.

**Review by kckeiks — APPROVED** [review]

**Comment by ozwaldorf** [comment]
It's essentially restoring the location where the bootstrap is called to how it was before when kad was in `discovery.rs`, where creating the new behavior calls bootstrap() instead of later on after some time in `service.rs`

**Review by b0xtch — APPROVED** [review]
